crates/masonry/src/testing/harness.rs

```diff
@@ -238,7 +243,7 @@ impl TestHarness {
     /// Create a bitmap (an array of pixels), paint the window and return the bitmap as an 8-bits-per-channel RGB image.
     pub fn render(&mut self) -> RgbaImage {
         let (scene, _tree_update) = self.render_root.redraw();
-        if std::env::var("SKIP_RENDER_TESTS").is_ok_and(|it| !it.is_empty()) {
+        if !check_env_enabled("ENABLE_RENDER_TESTS") {
```

Comment:
So as it happens, the render tests mostly work fine

So I'd probably revert this half of the change